From a32d3483e810e9025c7bcee2e6dce645e3f42fb1 Mon Sep 17 00:00:00 2001
From: Aichun Shi <aichun.shi@intel.com>
Date: Mon, 14 Nov 2022 10:35:25 +0800
Subject: [PATCH] Enable Intel Emmitsburg pinctrl for default config
Patch-mainline: Not yet, from openEuler
References: bsn#22
openEuler-commit: a32d3483e810e9025c7bcee2e6dce645e3f42fb1
Modified-by-SEL: No


Intel inclusion
category: feature
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I610P3
CVE: NA

Intel-SIG: openeuler_defconfig: Enable configs for Intel Emmitsburg pinctrl

--------------------------------------------

Enable necessary kernel configs for Intel Emmitsburg pinctrl in
openeuler_defconfig.

Signed-off-by: Aichun Shi <aichun.shi@intel.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 arch/x86/configs/openeuler_defconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 5e731188ab57..f9c61742607c 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -3750,7 +3750,7 @@ CONFIG_PINCTRL_BROXTON=m
 CONFIG_PINCTRL_CANNONLAKE=m
 CONFIG_PINCTRL_CEDARFORK=m
 CONFIG_PINCTRL_DENVERTON=m
-# CONFIG_PINCTRL_EMMITSBURG is not set
+CONFIG_PINCTRL_EMMITSBURG=m
 CONFIG_PINCTRL_GEMINILAKE=m
 CONFIG_PINCTRL_ICELAKE=m
 # CONFIG_PINCTRL_JASPERLAKE is not set
-- 
2.33.0

